src/js/components/CheckBox/stories/Simple.js

```diff
@@ -14,10 +14,12 @@ const SimpleCheckBox = ({ checked: checkedProp, ...rest }) => {
 };
 
 export const Simple = () => (
-  <>
+  <Grommet
```

Comment:
No need to modify this story. The concept is sufficiently demonstrated in other stories.